docs/faq.md

```diff
@@ -156,3 +174,26 @@ argocd ... --insecure
 
 Most likely you forgot to set the `url` in `argocd-cm` to point to your ArgoCD as well. See also
 [the docs](/operator-manual/user-management/#2-configure-argo-cd-for-sso).
+
+## Why is are resources of type `SealedSecret` stuck in the `Progressing` state?
+
+The controller of the `SealedSecret` resource may expose the status condition on resource it provisioned. Since
+version `v1.9.0` ArgoCD picks up that status condition to derive a health status for the `SealedSecret`.
+
+The versions `v0.12.0` to `v0.15.0` of the `SealedSecret` controller are affected by an issue regarding this status
```

Comment:
I think v0.15.0 is not yet released, and it will actually contain the fix? So maybe a better phrasing would be `Sealed secret versions before v0.15.0 ...`